content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -212,6 +212,11 @@ Certificates` section. Then, click :guilabel:`New` to configure the certificate:
    :alt: Digital certificate configuration.
    :align: center
 
+.. warning::
+   If the :guilabel:`Certificate Owner` field is set for a specific user and no certificate 
+   is shared with all users, the automatic sending of electronic documents and receipt 
+   acknowledgments will be **disabled**.
```

Comment:
Just a slight reword to avoid using "will"
```suggestion
   If the :guilabel:`Certificate Owner` field is set for a specific user, and there are no
   certificates shared with users, then the automatic sending of electronic documents and receipt
   acknowledgments is **disabled**.
```